public/datasets/global_happiness_raw.json

```diff
@@ -1,4 +1,15 @@
 {
+  "card": {
+    "description": "2019 Global Happiness report rankings, where countries are rated on different attributes that contribute to a total happiness score then ranked. The features in this dataset are taken from the years 2008 to 2018, which were averaged together for the report to create scores. This 'raw' data contains more rows than the 'Global Happiness 2019' dataset, but may give different results.",
```

Comment:
Do we want to keep in the mention of the other 'Global Happiness' dataset if it's no longer in the tool? Wouldn't want to send students on a goose chase.